test/acceptance/features/steps/steps.py

```diff
@@ -345,19 +359,31 @@ def parse_nullable_string(text):
 
 
 # STEP
-@step(u'Secret "{secret_name}" contains "{secret_key}" key with value "{secret_value:NullableString}"')
-def check_secret_key_value(context, secret_name, secret_key, secret_value):
+@step(u'Secret contains "{secret_key}" key with value "{secret_value:NullableString}"')
+def check_secret_key_value(context, secret_key, secret_value):
     openshift = Openshift()
+    time.sleep(5)
+    secret_name = ""
+    sbr = ServiceBinding()
+    if "sbr_name" in context:
+        json_path = "{.status.secret}"
+        secret_name = sbr.get_servicebinding_info_by_jsonpath(context.sbr_name, context.namespace.name, json_path)
```

Comment:
It looks that this code repeats on few places. Instead copy/paste, wrap it in a function and use it where needed.

Reply:
Done here 
54698bb